conda_forge_tick/audit.py

```diff
@@ -36,41 +38,78 @@ def audit_feedstock(fctx: FeedstockContext, ctx: MigratorSessionContext):
     return deps
 
 
+def grayskull_audit_feedstock(fctx: FeedstockContext, ctx: MigratorSessionContext):
+    """Uses grayskull to audit the requirements for a python package
+    """
+    # TODO: come back to this, since CF <-> PyPI is not one-to-one and onto
```

Comment:
This is a tricky issue. I think we need to render the recipe and extract the url. Then we can pull out the pypi name.

Reply:
maybe? I would be interested in actually having metadata that states what the name of the package is in other package managers